task/01-strings-tasks.js

```diff
@@ -83,9 +84,8 @@ function extractNameFromTemplate(value) {
  *   'John Doe'  => 'J'
  *   'cat'       => 'c'
  */
-function getFirstChar(value) {
-    throw new Error('Not implemented');
-}
+let getFirstChar = (value) => value.match(/^./g)[0];
```

Comment:
You shouldn't change the definition of functions, just do all changes inside already exists functions